packages/babel-traverse/src/path/inference/index.ts

```diff
@@ -23,7 +23,7 @@ const typeAnnotationInferringNodes = new WeakSet();
  * todo: split up this method
  */
 
-export function _getTypeAnnotation(): ?Object {
+export function _getTypeAnnotation(): any | undefined | null {
```

Comment:
`undefined | null` can be removed since `any` already contains all types

Reply:
indeed, it can be removed - will update the PR

(adding special case for nullable `any` in flow to typescript tool, so it would be updated in other places as well)